tests/support/foo-directories/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"version": "1.6.2",
+	"compilerOptions": {
+		"target": "es5",
+		"module": "umd",
+		"jsx": "react",
```

Comment:
I have no problem with adding an example that specifically tests the React/JSX/TSX approach, but it seems like this example may be conflating the directory issue and adding in some tsx specific details?